clients/src/main/java/org/apache/kafka/common/errors/OutdatedBrokerException.java

```diff
@@ -13,18 +13,18 @@
 package org.apache.kafka.common.errors;
 
 /**
- * Indicates that a request cannot be completed because an obsolete broker
+ * Indicates that a request cannot be completed because an outdated broker
  * does not support the required functionality.
```

Comment:
That is a good point.  We should have a separate exception for when the client is outdated.